templates/includes/records/archive-additional-resources.html

```diff
@@ -6,7 +6,6 @@ <h2>Accessions</h2>
         <form action="" id="archive" method="post">
             <label class="archive-additional-resources__label" for="accessionYear">Select a year</label>
             <select class="archive-additional-resources__select" name="accessionYear" onchange="updateAction()" id="accessionYear">
-                <option value="">--Please choose an option--</option>
```

Comment:
What are the UI implications here? Won't the first year be selected by default? Is that necessarily desirable?

Reply:
Adding a `required` attribute to the `select` element should prevent form submission if the default is selected

Reply:
No JS 👍   Thanks @ababic

Reply:
The update is good for now. Thanks.